rnd/autogpt_server/autogpt_server/executor/manager.py

```diff
@@ -160,27 +173,67 @@ def register_next_execution(node_link: Link) -> NodeExecution | None:
                 )
             )
 
+        # Complete missing static input pins data using the last execution input.
```

Comment:
Main change 1:
So now, whenever we want to try enqueue execution, we need to check if there are static input links/pins.
If there is, then we can use the latest execution result output for that pin (if there is any)